src/sandbox/KeywordFinder.java

```diff
@@ -0,0 +1,87 @@
+package sandbox;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class KeywordFinder {
+
+  public static final Logger logger = LogManager.getLogger(KeywordFinder.class);
```

Comment:
logger should be private. 
Please change the name of the variable to LOGGER as this is a static final field.